CHANGELOG.md

```diff
@@ -8,6 +8,9 @@
 * Feat: Support envelope based transport for events (#391)
   * The method signature of `Transport` changed from `Future<SentryId> send(SentryEvent event)` to `Future<SentryId> send(SentryEnvelope envelope)`
 * Remove `Sentry.currentHub` (#490)
+* Ref: Rename `cacheDirSize` to `maxCacheItems` and add `maxCacheItems` for iOS (#495)
+* Ref: `autoSessionTrackingIntervalMillis` renamed to `autoSessionTrackingInterval` (#495)
+* Ref: `anrTimeoutIntervalMillis` renamed to `anrTimeoutInterval` (#495)
```

Comment:
all the other SDKs have a simple Long as an interval in Millis.
not all the languages have a Duration builtin, not on Java 6/7 which is available on Android, for example, keeping it as a Long makes it easier to unify the options.
By comparing on Android, just noticed this issue https://github.com/getsentry/sentry-java/issues/1530

I'd not do this on this PR as it's not described in the 2 related issues.
I'll bring this up though in the unified API meeting, maybe more people would be interested in changing this as well.

Reply:
At least C# uses `TimeSpan`s for time units too: https://github.com/getsentry/sentry-dotnet/blob/main/src/Sentry/SentryOptions.cs#L276

Reply:
Yep, I'll do another PR/issue for that though.

Reply:
mmm, interesting, TIL, I will bring this to the meeting too then, thanks

Reply:
the downside of using such builtin durations as well is when reading configuration from external resources eg file system (spring boot etc), we'd need to fields to set such property, one with the duration and another with the duration type (millis, seconds, or whatever the user wanted to set).
we can discuss this in the other issue though.